src/modules/UsersRoutines.pm

```diff
@@ -98,24 +111,49 @@ sub CreateHome {
 	return 0;
     }
 
-    # if skeleton does not exist, do not copy it
-    if ($skel eq "" || !%{SCR->Read (".target.stat", $skel)}) {
-	if (! SCR->Execute (".target.mkdir", $home)) {
-	    y2error ("error creating $home");
-	    return 0;
-	}
+    # Create the home as btrfs subvolume
+    if ($use_btrfs) {
+        if (!valid_btrfs($home)) {
```

Comment:
Maybe this validation is not really needed since the `btrfs subvolume create /whatever/do/you/want` will produce a _stderr_ message similar to 

```
ERROR: not a btrfs filesystem: /whatever
```

What do you think?